test/distributed/org/apache/cassandra/distributed/test/log/ModelState.java

```diff
@@ -28,8 +28,7 @@
 import java.util.Set;
 import java.util.TreeMap;
 
-import org.apache.cassandra.harry.sut.TokenPlacementModel;
-import org.apache.cassandra.harry.sut.TokenPlacementModel.DCReplicas;
+import static org.apache.cassandra.harry.model.TokenPlacementModel.*;
```

Comment:
can you revert the `.*`?  goes against style guide

Reply:
rereading `[DISCUSSION] Cassandra's code style and source code analysis` to make sure I am not going crazy... I checked the link you sent and you are right we don't document this..

Reply:
spoke in slack, there is nothing merged to our style guide that says this, but I have had several conversations about this, so trying to track those down to see what happened.... best I see in that ML is that people are on-board but it looks like it just got dropped, nothing blocked from happening as far as I can tell.

Reply:
I have reverted regardless, just to reduce the diff